doc/lz4_Frame_format.md

```diff
@@ -98,6 +101,9 @@ that all blocks were fully transmitted in the correct order and without error,
 and also that the encoding/decoding process itself generated no distortion.
 Its usage is recommended.
 
+The combined __EndMark__ and __Content Checksum__ fields might sometimes be
```

Comment:
fields might sometimes be referred as -> fields might sometimes be referred to as